HULKs/hulk
# Use binaries to represent nao and webots implementation

## Introduced Changes

Currently, the hulk binary crate splits the webots and nao implementation by features and a single binary.
Thus, only either nao or webots can be targeted.

This PR introduces an independent binary for each target (`crates/hulk/src/bin/[nao/webots].rs`).

## ToDo / Known Issues

none

## Ideas for Next Iterations (Not This PR)

none

## How to Test

Thankfully, we have a CI to check if everything still compiles.

To test the entire pipeline, use pepsi to upload to a nao (`pepsi upload xx`).

To test the webots binary, build, run and see if webots still executes correctly.
`pepsi run`
or
`pepsi build` and use the `hulk` controller in webots
